yaptide/routes/common_sim_routes.py

```diff
@@ -136,35 +133,28 @@ def post():
 
         make_commit_to_db()
         logging.debug("Marking simulation as completed")
-        update_dict = {
-            "job_state": EntityState.COMPLETED.value,
-            "end_time": datetime.utcnow().isoformat(sep=" ")
-        }
+        update_dict = {"job_state": EntityState.COMPLETED.value, "end_time": datetime.utcnow().isoformat(sep=" ")}
         update_simulation_state(simulation=simulation, update_dict=update_dict)
         return yaptide_response(message="Results saved", code=202)
 
     class APIParametersSchema(Schema):
         """Class specifies API parameters"""
 
         job_id = fields.String()
+        estimator_name = fields.String(load_default=None)
 
     @staticmethod
-    @requires_auth()
-    def get(user: UserModel):
-        """Method returning job status and results"""
-        schema = ResultsResource.APIParametersSchema()
-        errors: dict[str, list[str]] = schema.validate(request.args)
-        if errors:
-            return yaptide_response(message="Wrong parameters", code=400, content=errors)
-        param_dict: dict = schema.load(request.args)
+    def _get_single_estimator(sim_id, estimator_name):
```

Comment:
add type hinting

Reply:
by the way - is there any reason for this method to be a class method ? Can't it be a plain python method outside a class ?